src/components/SubmissionDetailsPage/SubmissionDetailsPage.js

```diff
@@ -65,7 +28,7 @@ const SubmissionDetailsPage = ({ user }) => {
     <Modal name={data.getSubmission.name} setShowModal={setShowModal} id={id}/>
   
   if (loading) return <p>loading...</p>
```

Comment:
Did we want to use the `<UploadingSpinner/>` here as well?

Reply:
Good catch, I'd say yes! I think we should use it anywhere we have a loading gql request